aspnetcore/docfx.json

```diff
@@ -42,7 +42,11 @@
       "_navRel": "/foo",
       "searchScope": [
         "ASP.NET Core"
-      ]
+      ],
+      "ms.topic": "article",
```

Comment:
you should now change to conceptual if you wanna add this in the global settings 😄